py/sight/widgets/decision/decision.py

```diff
@@ -678,6 +705,108 @@ def get_decision_outcome_proto(outcome_label: str,
   return decision_outcome
 
 
+def get_decision_outcome_proto_from_cached(outcome_label: str,
+                                           decision_message: DecisionMessage):
+
+  logging.info('decision message =>%s', decision_message)
+
+  decision_outcome = sight_pb2.DecisionOutcome(outcome_label=outcome_label)
+  decision_outcome.reward = decision_message.reward
+  decision_outcome.outcome_params.CopyFrom(
+      convert_dict_to_proto(dict=decision_message.outcome_params))
+  decision_outcome.discount = decision_message.discount
+  return decision_outcome
+
+
+def _configure_client_and_worker(sight):
+  """Configures the client and worker identifiers."""
+  if FLAGS.deployment_mode in ["local"] or _TRAINED_MODEL_LOG_ID.value:
+    global _sight_id
```

Comment:
Do we need this global _sight_id variable? It breaks modularity and I don't see it being used anywhere.

Reply:
removing _sight_id as it not being used anywhere